front-end/App.js

```diff
@@ -24,12 +48,5 @@ export default function App() {
     loadExpoFonts()
   })
```

Comment:
@Jeffzholy actually let's refactor this into our custom `useFonts` hook:
https://reactjs.org/docs/hooks-custom.html